lib/BaseSlackHandler.js

```diff
@@ -48,15 +48,15 @@ BaseSlackHandler.prototype.getSlackRoomNameFromID = async function(id, token) {
     return id;
 };
 
-BaseSlackHandler.prototype.replaceChannelIdsWithNames = async function(message, token) {
+BaseSlackHandler.prototype.replaceChannelIdsWithNames = async function(message, text, token) {
```

Comment:
is message still used? Also you are returning message but editing text?

Reply:
Yeah, message is being used to lookup teams and senders and stuff. It should be returning text.